doc/user/content/administration/billing.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Usage & Billing"
+description: "Provides an overview of various factors that can affect cost."
+menu:
+  main:
+    parent: "administration"
+    weight: 10
+---
+
+From the [console](https://console.materialize.com/) (`Admin` > `Usage &
```

Comment:
```suggestion
From the [Materialize console](https://console.materialize.com/) (`Admin` > `Usage &
```

Reply:
Can't remember where we discussed this, but "Materialize console" is what the docs consistently use.